lib/utils/registry-secrets.ts

```diff
@@ -0,0 +1,65 @@
+import * as ajv from 'ajv';
```

Comment:
This should include the Balena boilerplate (technically it all should, but I'm aware the other files you're just appending to).

Reply:
Added, thanks.